bioverse/functions.py

```diff
@@ -882,6 +918,99 @@ def compute_habitable_zone_boundaries(d):
 
     return d
 
+def compute_habitable_zone_boundaries_all_stars(d):
```

Comment:
there are now two functions that compute HZ boundaries. They have the same docstring and they share a lot of code (all but 18 lines). Do we need both? Can they be combined?